specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2021-03-01/storagecache.json

```diff
@@ -1795,6 +1743,28 @@
         "statusDescription": {
           "description": "Describes explanation of state.",
           "type": "string"
+        },
+        "conditions": {
+          "description": "Outstanding conditions that need to be investigated and resolved.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Condition"
+          }
+        }
+      }
+    },
+    "Condition": {
+      "type": "object",
+      "description": "Outstanding conditions that will need to be resolved.",
+      "properties": {
+        "timestamp": {
```

Comment:
Sounds like timestamp and message are readonly properties? If yes, please mark them as readonly.

Reply:
Updated in latest commit.